src/components/ebay-menu/template.marko

```diff
@@ -33,9 +33,9 @@
                 w-onkeydown="handleItemKeydown"
                 w-preserve-attrs="tabindex,data-makeup-index"
                 ${item.htmlAttributes}>
-                <span w-preserve-body>
+                <span>
                     <span body-only-if(!item.badgeNumber) aria-hidden='true'>
-                        <span body-only-if(true) w-body=item.renderBody/>
+                        <span w-body=item.renderBody/>
```

Comment:
I thought we didn't want this span in the output, has this changed?

Reply:
Unless you know another way, the only way I found to preserve that content was to remove this along with the `w-preserve-body` on the outer `<span>`.

Reply:
Yes, I'd prefer not to have this span, though.

Reply:
Okay, if the `body-only-if` true is messing it up then I'd say lets go with this for now.